code/modules/mod/mod_theme.dm

```diff
@@ -507,14 +494,14 @@
 	armor = list(MELEE = 10, BULLET = 5, LASER = 5, ENERGY = 5, BOMB = 10, RAD = 0, FIRE = 75, ACID = 150)
 
 /datum/mod_theme/rescue
-	name = "rescue"
-	desc = "An advanced version of DeForest Medical Corporation's medical suit, designed for quick rescue of bodies from the most dangerous environments."
-	extended_desc = "An upgraded, overtuned version of DeForest Medical Corporation's medical suit, with BioTech Solutions making heavy modifications. \
-		designed for quick rescue of bodies from the most dangerous environments. The same advanced leg servos \
-		as the base version are seen here, giving paramedics incredible speed, but the same servos are also in the arms. \
-		Users are capable of quickly hauling even the heaviest crewmembers using this suit, \
-		all while being entirely immune against chemical and thermal threats. \
-		It is slightly more demanding of power than civilian-grade models, and weak against fingers tapping the glass."
+	name = "'Valkyrie' rescue"
+	desc = "An next-generation medical suit for casualty care in even the most hostile of conditions. Fast, chemically sealed, and decently armored to boot."
+	extended_desc = "A direct upgrade to the older Apollo line, the D-112 'Valkyrie' is an advanced medical modsuit produced by the DeForest Medical Corporation. Offering iterative improvement over earlier lines, \
+		the Valkyrie uses improved servo systems and drive motors to deliver faster movement speed than the Apollo line, while a high-efficiency energy core significantly reduces the high power draw of the line. \
```

Comment:
```suggestion
		the Valkyrie uses improved servo systems and drive motors to deliver faster movement speed than the Apollo line, and it comes standard with a high-efficiency energy core which helps offset its otherwise outsized power draw. \
```